src/services/type.ts

```diff
@@ -0,0 +1,11 @@
+export type PublicReleaseMessage = {
```

Comment:
Lets be more specific with the file naming so that we don't end up with a generic "types" file that is a dumping ground for a bunch of stuff that is hard to keep track of...

How about a `src/types/kafkaMessages.ts` file?